importer/api/__init__.py

```diff
@@ -130,6 +131,10 @@ def import_course(bundle, repo_id, user_id, static_dir):
     import_static_assets(course, static_dir)
     import_children(course, src, None, '')
     populate_xanalytics_fields.delay(course.id)
+    # This triggers a bulk indexing of all LearningResource instances
```

Comment:
The old code is still doing the non-bulk indexing, so sadly you won't be able to watch it run faster. However, I did add statsd decorators, so it should still be possible to evaluate.

The new code does _only_ the bulk indexing for newly-created items (see `search/signals.py`  function `handle_resource_update`, which quits if the `created` flag is `True`). Then, individual indexing is only done upon update (by that `handle_resource_update` function).